PV260_Project/PresentationLayer/UI/DataSetUi.cs

```diff
@@ -118,5 +118,40 @@ public class DataSetUi : BaseUi, IDataSetUi
                 }
             }
         }
+        
+        private async Task SendNotification() {
+            var emails = await _userEmailService.GetAllRegisteredEmails();
+
+            if (!emails.Any())
+            {
+                Console.WriteLine("Notifications were not sent, because there aren't any registered emails.");
+                Console.WriteLine("Please, register at least one email in order to send notifications.");
+            }
```

Comment:
I would `return` if there are no emails, because there is no reason to stay in this part anyway and the the `else` can be removed.